moveit_setup_assistant/src/widgets/header_widget.cpp

```diff
@@ -171,15 +173,15 @@ void LoadPathWidget::btn_file_dialog()
 // ******************************************************************************************
 // Get the QString path
 // ******************************************************************************************
-const QString LoadPathWidget::getQPath()
+QString LoadPathWidget::getQPath() const
 {
   return path_box_->text();
 }
 
 // ******************************************************************************************
 // Get Std String path
 // ******************************************************************************************
-const std::string LoadPathWidget::getPath()
+std::string LoadPathWidget::getPath() const
```

Comment:
my original return value seems wrong but it should be ``const std::string&`` right?

Reply:
Yes, that's another option, which doesn't copy the string.

Reply:
No, in this case we actually forward the return value of a function. `const reference` only makes sense when return internal variables.